device-types/Sophos/XGS-3300.yaml

```diff
@@ -0,0 +1,55 @@
+---
+manufacturer: Sophos
+model: XGS 3300
+slug: sophos-xgs-3300
+u_height: 1
+is_full_depth: false
+subdevice_role: parent
+airflow: front-to-rear
```

Comment:
Hello, thank you for your contribution! In an attempt to have the most accurate and complete data we can, I have a few additional data points we would like you to add if possible.

1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. Often these are listed in the datasheet, but sometimes they are not

If you have any questions, please let us know :)

Reply:
added weight and _unit from datasheet like in other device-types, not imported in our instance, but that seems to be an open issue.